homeassistant/components/google/calendar.py

```diff
@@ -231,7 +233,7 @@ def __init__(
         self._search: str | None = data.get(CONF_SEARCH)
         self._ignore_availability: bool = data.get(CONF_IGNORE_AVAILABILITY, False)
         self._event: CalendarEvent | None = None
-        self._name: str = data[CONF_NAME]
+        self._attr_name = data[CONF_NAME].lower().capitalize()
```

Comment:
I think `lower` is redundant here. `capitalize` should take care of it automatically.
```suggestion
        self._attr_name = data[CONF_NAME].capitalize()
```